content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,131 @@
+---
+title:       "Newsletter 102 - 2022/2023 news"
+author:      "GeoB99"
+date:        2023-03-25
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers and enthusiasts! It has been quite a long time since we published the 101th newsletter and so far no further updates have been posted since then.
+While the [ReactOS Twitter account](https://twitter.com/reactos) does provide announcements, posts about working applications and such from time to time, much of what is happening with the project
+as a whole isn't mentioned. Most of what we are going to talk about is the current situation with releases and overall ReactOS development.
+
+## Why does it take so long to release a new version of ReactOS? Are you guys dead?
+The latest release of ReactOS is 0.4.14, published on 16th of December, 2021. That release alone took a year to be released. Back then the ReactOS followed the 3-months cadence, which every 3 months
```

Comment:
For these markdown files, it's better to keep one sentence per line (so applying suggestions is easier), and don't include artificial newlines (they will be stripped by hugo anyway).

Paragraphs will be inserted where there are multiple newlines.

Reply:
![image](https://github.com/reactos/web-content/assets/610685/23adbace-8ffb-4730-83ba-b66832e5cedf)